src/ReflectionMethod.php

```diff
@@ -100,23 +103,25 @@ public function __toString()
         $methodParameters = $this->getParameters();
         try {
             $prototype = $this->getPrototype();
-        } catch (\ReflectionException $e) {
+        } catch (BaseReflectionException) {
             $prototype = null;
         }
         $prototypeClass = $prototype ? $prototype->getDeclaringClass()->name : '';
 
         $paramString = '';
         $indentation = str_repeat(' ', 4);
         foreach ($methodParameters as $methodParameter) {
-            $paramString .= "\n{$indentation}" . $methodParameter;
+            $paramString .= "\n$indentation" . $methodParameter;
         }
 
+        /** @noinspection PhpFormatFunctionParametersMismatchInspection */
         return sprintf(
-            "%sMethod [ <user%s%s%s>%s%s%s %s method %s ] {\n  @@ %s %d - %d{$paramFormat}{$returnFormat}\n}\n",
+            "%sMethod [ <user%s%s%s>%s%s%s %s method %s ] {\n  @@ %s %d - %d$paramFormat$returnFormat\n}\n",
             $this->getDocComment() ? $this->getDocComment() . "\n" : '',
-            $prototype ? ", overwrites {$prototypeClass}, prototype {$prototypeClass}" : '',
+            $prototype ? ", overwrites $prototypeClass, prototype $prototypeClass" : '',
             $this->isConstructor() ? ', ctor' : '',
-            $this->isDestructor() ? ', dtor' : '',
+            // $this->isDestructor() ? ', dtor' : '',
```

Comment:
This line can be removed at all, as PHP8 doesn't report this `dtor` suffix anymore https://3v4l.org/G1hcX

Reply:
https://github.com/goaop/parser-reflection/pull/122/commits/c331ba429e241fa8424fe59cccefdf3bff5f607e